modules/web-console/frontend/app/components/bs-select-menu/template.pug

```diff
@@ -37,11 +37,11 @@ ul.bs-select-menu(
             ng-click='$select($index, $event); $event.stopPropagation();'
             ng-class=`{ 'bssm-item-button__active': $isActive($index) }`
             data-placement='right auto'
-            title='{{ ::match.label | bsSelectStrip }}'
```

Comment:
If "sce in chart menu" approach works, consider reverting all unnecessary changes to bs-select-menu.